app/src/main/java/com/khmelenko/lab/travisclient/adapter/JobsListAdapter.java

```diff
@@ -60,8 +60,7 @@ public void onBindViewHolder(JobViewHolder holder, int position) {
 
                 Drawable drawable = BuildStateHelper.getBuildImage(state);
                 if (drawable != null) {
-                    drawable.setBounds(0, 0, 30, 30);
```

Comment:
setCompoundDrawablesWithIntrinsicBounds sets the bounds automatically based on the drawables intrinsic bounds (16dp for the status icons).
I tested it on both of my test devices (Galaxy S2 with API 16 and Nexus 6P with API 25) and it worked as expected.